addons/l10n_in_tcs_tds/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_in.cess_sale_5" model="account.tax.template">
```

Comment:
would not it be easier to do this in l10n_in right away to avoid again having to put it in post_init_hook?

Reply:
yes, I move this to l10n_in